### PR TITLE
ci - build bas-platform

### PR DESCRIPTION
**Pull request by tomer-epstein**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,10 +62,30 @@ jobs: # a collection of steps
...
+          working_directory: bas-platform
+          command: npm run compile
+      - run:
+          name: create .vsix
+          working_directory: bas-platform
+          command: npx vsce package .
+      # - run:
+      #     name: test bas-platform
+      #     working_directory: bas-platform
+      #     command: npm run test
```
Comment (perspectivus1) [inline .circleci/config.yml:80]: Is this commented on purpose because we don't have tests yet?

Comment (tomer-epstein) [reply .circleci/config.yml:80]: Yes, I'll add an issue to implement tests for bas_platform

File: .circleci/config.yml [hunk 2]
```diff
@@ -62,10 +62,30 @@ jobs: # a collection of steps
...
+      - store_artifacts: # special step to save test results as as artifact
+          path: bas-platform/coverage
```
Comment (perspectivus1) [inline .circleci/config.yml:82]: Won't this fail if this path doesn't exist?

Comment (tomer-epstein) [reply .circleci/config.yml:82]: passing.

**Review by perspectivus1 — APPROVED** [review]

**Pull request merged**